### PR TITLE
Create and update profile

### PR DESCRIPTION
**Pull request by KorkuTegbe**
profile controller for creating user profiles and updating profile. Image upload included.

### CODE REVIEW

File: controllers/profileController.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+cloudinary.config({
+    cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_SECRET
+})
+
+
+const createProfile = catchAsync(async (req,res) => {
+    // get req body and files
```
Comment (ademesojosiah) [inline controllers/profileController.js:12]: You and this you catchAsync you sha no wan use ‘express-async-error’ sha come teach me

File: routes/profileRoute.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const router = require('express').Router()
+const profileController = require('../controllers/profileController')
+const multer = require('multer')
+const upload = multer({ dest: 'uploads/'})
```
Comment (ademesojosiah) [inline routes/profileRoute.js]: my oga , let multer know that you want only png,jpg files
you can check the post route

File: controllers/profileController.js (hunk 1 above)
Comment (KorkuTegbe) [reply controllers/profileController.js:12]: lmao. i no sabi this express-async-error. I go start use am, no worry,

File: routes/profileRoute.js (hunk 2 above)
Comment (KorkuTegbe) [reply routes/profileRoute.js]: alright sure. thanks. I think I'll create a multer config file and then import it where multer is needed.

File: controllers/profileController.js (hunk 1 above)
Comment (KorkuTegbe) [reply controllers/profileController.js:12]: I got rid of catchAsync and used 'express-async-error'. Check it out

**Comment by KorkuTegbe** [comment]
- Configured cloudinary and multer(file size limit, type and location) in different files to be imported wherever they might be needed. 
- Used `express-async-error` instead of `catchAsync` in profile controller. Special acknowledgement to Oga Josiah

**Pull request merged**